src/views/Home.js

```diff
@@ -0,0 +1,48 @@
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import '../styles/home.css';
+import Hero from '../components/Hero';
+import Card from '../components/Card';
+import Nav from '../components/Nav';
+
+const Home = () => {
+  const countries = useSelector(({ countriesReducer }) => countriesReducer);
+  const covid = useSelector(({ covidReducer }) => covidReducer);
+  const arr = [...new Set(countries.map((c) => c.region))].sort();
+
```

Comment:
- [OPTIONAL] Please  it will be nice to add a component to filter the list of items displayed in the home page by some parameter(`e.g. a search field to display the items based on the number of cases.`).